README.md

```diff
@@ -4,6 +4,15 @@
 
 The open-source RChain project is building a decentralized, economic, censorship-resistant, public compute infrastructure and blockchain. It will host and execute programs popularly referred to as “smart contracts”. It will be trustworthy, scalable, concurrent, with proof-of-stake consensus and content delivery.
 
+### Communication
+
+The `comm` subproject contains code for network related operations for RChain.
+
+The network layer is the lowest level component in the architecture and it
+is featured in our **Node.Hello (v0.1) release**. The simplest way to get
+started is with docker: `docker run -ti rchain/rchain-comm`. For other options,
+see `README.md` under `comm`.
```

Comment:
This is great. Let's cram some links in like:
. . . is with [docker](https://store.docker.com/community/images/rchain/rchain-comm): `docker` . . .
. . . see [comm/README.md](https://github.com/rchain/rchain/tree/master/comm).